examples/prompts/llm_monitoring/openai_proxy_quickstart.ipynb

```diff
@@ -241,64 +250,25 @@
    },
    "outputs": [],
    "source": [
-    "# Replace WB_ENTITY, WB_PROJECT, and STREAM_NAME to specify where to store the logs.\n",
-    "# You will likely get an authentication error if you don't do this.\n",
-    "\n",
     "%%bash\n",
-    "curl \"https://api.wandb.ai/proxy/openai/v1/chat/completions\" \\\n",
+    "curl \"https://proxy.wandb.ai/proxy/openai/v1/chat/completions\" \\\n",
     "-H \"Authorization: Bearer $WANDB_API_KEY:$OPENAI_API_KEY\" \\\n",
     "-H \"Content-Type: application/json\" \\\n",
-    "-H \"X-Wandb-Entity: WB_ENTITY\" \\\n",
-    "-H \"X-Wandb-Project: WB_PROJECT\" \\\n",
-    "-H \"X-Wandb-Stream: STREAM_NAME\" \\\n",
+    "-H \"X-Wandb-Entity: stacey\" \\\n",
+    "-H \"X-Wandb-Project: proxy_test\" \\\n",
+    "-H \"X-Wandb-Stream: proxy_stream\" \\\n",
     "-H \"X-Wandb-Client-Id: Shakespearean-Gradients-001\" \\\n",
     "-H \"X-Wandb-Attribute-Persona: Shakespeare\" \\\n",
     "-d '{\n",
     "     \"model\": \"gpt-3.5-turbo\",\n",
     "     \"messages\": [{\"role\": \"user\", \"content\": \"Tell me a joke about loss functions!\"}]\n",
     "   }'"
    ]
-  },
-  {
```

Comment:
why remove?

Reply:
https://weightsandbiases.slack.com/archives/C03BSTEBD7F/p1695923749471649?thread_ts=1695922367.393609&cid=C03BSTEBD7F

Reply:
the issue is we have a check in weave for `WANDB_API_KEY` being set AND `.netrc` exists. However, for proxy when netrc exists and they use `proxy.wandb.ai`, they need `WANDB_API_KEY` or else there is no auth. But weave prevents this request from going through.

Reply:
why do they have netrc in their colab?!